aws-serverless-java-container-core/pom.xml

```diff
@@ -105,9 +122,14 @@
         <dependency>
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-web</artifactId>
-            <version>5.7.7</version>
+            <version>6.0.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.activation</groupId>
+            <artifactId>jakarta.activation-api</artifactId>
+            <version>2.1.0</version>
```

Comment:
2.1.1? Why is it needed to have this with compile scope?

Reply:
It's used in the AwsServletContext class. With a test scope, it gives the following errors:
 ```
package jakarta.activation.spi does not exist
cannot find symbol
[ERROR]   symbol:   class MimetypesFileTypeMap
[ERROR]   location: class com.amazonaws.serverless.proxy.internal.servlet.AwsServletContext

```
Updated to 2.1.1